proto/v1/activity_service.proto

```diff
@@ -0,0 +1,213 @@
+syntax = "proto3";
+
+package bytebase.v1;
+
+import "google/api/annotations.proto";
+import "google/api/client.proto";
+import "google/api/field_behavior.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/wrappers.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/struct.proto";
+
+import "v1/common.proto";
+
+option go_package = "generated-go/v1";
+
+service ActivityService {
+  rpc CreateActivity(CreateActivityRequest) returns (Activity) {
```

Comment:
We don't need Create/UpdateActivity.

Reply:
Our issue comment store in activity. Do you mean we should use `CreatIssueComment` and `UpdateIssueComment`?